modules/xmpp/ChatRoom.js

```diff
@@ -982,7 +900,6 @@ export default class ChatRoom extends Listenable {
      * rejected.
      */
     leave () {
-        this._dispose();
```

Comment:
It is done by JitsiConference which holds the instance:
https://github.com/jitsi/lib-jitsi-meet/pull/372/files#diff-40e35c919311b508dc5b45a7bf028f54R189